ament_flake8/test/test_flake8.py

```diff
@@ -15,9 +15,7 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
-
-from ament_flake8.main import main  # noqa
+from ament_flake8.main import main
```

Comment:
@Arnatious This change resulted in two linter warnings about `os` and `sys` now being unused in this file: https://ci.ros2.org/view/nightly/job/nightly_linux_debug/1277/testReport/ament_flake8.test/test_flake8/test_flake8/

Please create a PR to address them.

Reply:
#178